packages/Adapters/Moab/example/interpolation.cpp

```diff
@@ -338,7 +344,19 @@ int main( int argc, char *argv[] )
 
     error_l2_norm = std::sqrt( error_l2_norm );
     tag_l2_norm = std::sqrt( tag_l2_norm );
-    std::cout << "|e|_2 / |f|_2: " << error_l2_norm / tag_l2_norm << std::endl;
+    double pass_criteria = error_l2_norm / tag_l2_norm;
+    std::cout << "|e|_2 / |f|_2: " << pass_criteria << std::endl;
+
+    std::cout << std::endl;
+    std::cout << "End Result: TEST ";
+    if ( pass_criteria < 1.0e-2 )
```

Comment:
The test function has absolute values in it so i suppose the first order discretization isn't doing as good of a job compared to the analytic result?